docker-compose.yml

```diff
@@ -34,6 +34,8 @@ services:
             service: mender-base
         networks:
             - mender
+        volumes:
+          - ./extra/integration-version.json:/var/www/mender-gui/dist/integration-version.json
```

Comment:
@GregorioDiStefano something like this?
Not sure where it should be mounted in the container though.